vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -1190,6 +1190,13 @@
   environment:
     SSHPASS: "{{ sftp_arkcase_password }}"
 
+- name: create libraries directory if it does not exist
+  become: yes
+  become_user: arkcase
+  file:
+    path: "{{ root_folder }}/data/arkcase-home/.arkcase/libraries"
```

Comment:
In this file, all other references to the .arkcase folder use `/home/arkcase/.arkcase`, which would be either a regular folder or a symlink to  `"{{ root_folder }}/data/arkcase-home/.arkcase`.  Let's use `/home/arkcase/.arkcase` here too, for consistency.